pkg/util/templates/templates.go

```diff
@@ -100,7 +99,7 @@ func sufficientProbes(windowSize, label string) string {
 }
 
 //render creates a monitoring rule for the defined multiwindow multi-burn rate alert
-func (r *multiWindowMultiBurnAlertRule) render(url, percent, label, alertName string) monitoringv1.Rule {
+func (r *multiWindowMultiBurnAlertRule) render(url, percent, label string, namespacedName types.NamespacedName, alertNameSuffix string) monitoringv1.Rule {
```

Comment:
these are a looot of parameters, you think we could move some of them to the rule type?

Reply:
Glad you mention it! Addressed in the new commit.